cloud/aws/templates/aws_oidc/variables.tf

```diff
@@ -547,6 +541,36 @@ variable "ecs_min_cpu_period" {
   default     = "60"
 }
 
+variable "ecs_task_memory" {
```

Comment:
I think maybe we should leave this where it was previously, so that it lives with the other ECS variables (and the alarm variables stay down here). Thoughts? (For both this file and the file above)

Reply:
ok!